### ISSUE TITLE
Docker image motoserver/moto:1.3.14 fails to start

### ISSUE THREAD

**Issue by stevenafleetwood** [spulec/moto#2579]
Throws following error when started:

```
File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 583, in _build_master

    ws.require(__requires__)

  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 900, in require

    needed = self.resolve(parse_requirements(requirements))

  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 791, in resolve

    raise VersionConflict(dist, req).with_context(dependent_req)

pkg_resources.ContextualVersionConflict: (python-dateutil 2.8.1 (/usr/lib/python3.6/site-packages), Requirement.parse('python-dateutil<2.8.1,>=2.1; python_version >= "2.7"'), {'botocore'})
```

This does not happen with `motoserver/moto:1.3.5`

I cannot use `motoserver/moto:1.3.5` for my testing as I need the changes made to support the shift from `ecr` to `api.ecr` fixed in https://github.com/spulec/moto/pull/2036

**Comment by fleetwoods-origami** [comment]
I built the docker image locally in a virtualenv environment and it starts up fine.

```python --version
Python 3.7.3```

```uname -a
Darwin ******** 19.0.0 Darwin Kernel Version 19.0.0: Thu Oct 17 16:17:15 PDT 2019; root:xnu-6153.41.3~29/RELEASE_X86_64 x86_64```

**Comment by stevenafleetwood** [comment]
I built the docker image locally in a virtualenv environment and it starts up fine.

`python --version: Python 3.7.3`

`uname -a: Darwin ******** 19.0.0 Darwin Kernel Version 19.0.0: Thu Oct 17 16:17:15 PDT 2019; root:xnu-6153.41.3~29/RELEASE_X86_64 x86_64`

**Comment by gruebel** [comment]
This was fixed with #2570
